### PR TITLE
add nil check if there is no commit message

### PR DESCRIPTION
**Pull request by anthonywoo**
**Note**: Samson is a public repo, do not include Zendesk-internal information, urls, etc.

Fix an issue where the deploys page breaks if a commit has no message

### References
- Jira link: 

### Risks
- Low

### CODE REVIEW

**Review by rsb — APPROVED** [review]

File: app/models/changeset/commit.rb [hunk 1]
```diff
@@ -28,7 +28,13 @@ def summary
...
 
   def summary_long
-    @data.commit.message.split("\n").first
+    summary = @data.commit.message.split("\n").first
+
+    if summary.nil?
+      return ""
+    end
+
+    summary
```
Comment (grosser) [inline app/models/changeset/commit.rb:37]:
```suggestion
    @data.commit.message.split("\n").first || ""
```

**Review by grosser — APPROVED** [review]

**Review by grosser — COMMENTED** [review]
test case would also be nice

File: app/models/changeset/commit.rb (hunk 1 above)
Comment (rsb) [reply app/models/changeset/commit.rb]: This has been updated to reflect your one liner.

File: .bundle/config [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_WITHOUT: "nothing:postgres"
```
Comment (grosser) [inline .bundle/config:2]: should be in your global .gitignore

Comment (rsb) [reply .bundle/config]: removed. Thanks

**Review by grosser — APPROVED** [review]

**Pull request merged**